js/join.js

```diff
@@ -244,6 +264,7 @@ function previewFiller() {
   previewFunFact.innerHTML = window.localStorage.getItem('funFact');
   previewWhyRds.innerHTML = window.localStorage.getItem('whyRds');
   previewHeardAbout.innerHTML = window.localStorage.getItem('foundFrom');
+  previewNumberOfHours.innerHTML = window.localStorage.getItem('numberOfHours');
```

Comment:
`innerHTML` would store the text with tags, E.g `<p><span>Hello</span></p>` here if we do `.innerHTML` on p tag it will return `<span>Hello</span>`. Are you sure we want this here, if you're only concerned with the text then consider using `textContent` or `innerText`